recipe/meta.yaml

```diff
@@ -21,13 +21,19 @@ requirements:
     - cross-python_{{ target_platform }}     # [build_platform != target_platform]
     - numpy                                  # [build_platform != target_platform]
     - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - {{ compiler('fortran') }}
```

Comment:
@jochym It seems we cannot provide fortran bindings on windows. I can't find the relevant jinja/environment variable to trigger this conditionally. Can you help with this?